packages/subscription-manager/src/components/SubscriptionManagerContainer/SubscriptionManagerContainer.tsx

```diff
@@ -12,18 +12,57 @@ export type SubscriptionManagerContainerProps = {
 	children?: React.ReactNode;
 };
 
+const getEmailAddress = () => {
```

Comment:
This is currently getting email from the cookie only, ignoring whether the user is logged in our not.